k8s/base.py

```diff
@@ -186,6 +186,13 @@ def delete(cls, name, namespace="default", **kwargs):
         url = cls._build_url(name=name, namespace=namespace)
         cls._client.delete(url, **kwargs)
 
+    @classmethod
+    def delete_list(cls, namespace="default", labels=None, **kwargs):
+        selector = ",".join("{}{}".format(k, v if isinstance(v, LabelSelector) else Equality(v))
```

Comment:
I'd consider the case of no value provided (selecting on existence of a key). e.g. if value is `None` then use `Exists`. I remember needing this as the selector `key != value` would be true (so would match all objects) if key doesn't exist.

Reply:
So the caller can already pass in `{"foo": Exists()}` if they want that - but you think it should be a special case? The logic here is taken from `find`; I'm not sure they should be different.

Reply:
ah right, that's possible so that's fine 👍